frontend/src/pages/simulationWindow.js

```diff
@@ -16,14 +15,14 @@ function SimulationWindow() {
 
   const [activePage, setActivePage] = useState("introduction");
   const [pages, setPages] = useState({
-    introduction: {visited: true, pageNumber: 1, html: (<Introduction/>)},
-    projectAssignment: {visited: false, pageNumber: 2, html: (<ProjectAssignment/>)},
-    responseOne: {visited: false, pageNumber: 3, html: (<ResponseOne/>)},
-    gatheredInformation: {visited: false, pageNumber: 4, html: (<GatheredInformation/>)},
-    stakeholders: {visited: false, pageNumber: 5, html: (<Stakeholders/>)},
-    results: {visited: false, pageNumber: 6, html: (<Results/>)},
-    feedback: {visited: false, pageNumber: 7, html: (<Feedback/>)},
-    response: {visited: false, pageNumber: 8, html: (<Response/>)}
+    introduction: {visited: true, completed: true, pageNumber: 1, html: (<Introduction/>)},
```

Comment:
I think this is supposed to be dynamic right? Like there can be multiple of certain components and the order can change. Could be wrong, but it would be worth asking the customer if you are unsure (unless this is just hardcoded for demo purposes)

Reply:
This is hard coded for demo purposes